gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/token/ApimlAccessTokenProvider.java

```diff
@@ -134,32 +158,21 @@ private String initializeSalt() throws CachingServiceClientException {
     }
 
     public String getToken(String username, int expirationTime, Set<String> scopes) {
-       expirationTime = Math.min(expirationTime, 90);
-       if (expirationTime <= 0) {
-           expirationTime = 90;
-       }
-       return authenticationService.createLongLivedJwtToken(username, expirationTime, scopes);
+        expirationTime = Math.min(expirationTime, 90);
```

Comment:
There won't be side effects, but re-assigning the parameter always makes me nervous.